packages/provider/package.json

```diff
@@ -13,9 +13,10 @@
   },
   "scripts": {
     "postinstall": "shx mkdir -p dist && link-module-alias",
-    "build": "tsc --build tsconfig.build.json && link-module-alias",
+    "build": "npm build:api && tsc --build tsconfig.build.json && link-module-alias",
     "build:watch": "tsc --build -w tsconfig.build.json && link-module-alias",
-    "clean": "tsc --build tsconfig.build.json --clean  && shx rm -rf coverage *.log junit.xml dist && jest --clearCache",
+    "build:api": "npx swagger-typescript-api --axios --single-http-client --modular -p https://api.evmos.org/openapi.json -o ./src/generated",
```

Comment:
I'm not sure what `  --disableProxy                disabled proxy (default: false)` option does.

https://github.com/acacode/swagger-typescript-api#-usage